bluepyopt/ephys/simulators.py

```diff
@@ -163,7 +170,10 @@ def run(
             rng.Random123_globalindex(random123_globalindex)
 
         try:
-            self.neuron.h.run()
+            self.neuron.h.finitialize()
```

Comment:
Are we sure this 'exactly' replicates the 'run()' behavior of Neuron? We don't want to change results if not necessary.